mllib/src/test/scala/org/apache/spark/ml/recommendation/ALSSuite.scala

```diff
@@ -520,6 +488,81 @@ class ALSSuite
   }
 }
 
+class ALSCleanerSuite extends SparkFunSuite {
+  test("Clean shuffles") {
```

Comment:
This is not about the style. It is not clear about what it does from the name, especially when the two tests are named "Clean shuffles" and "ALS shuffle cleanup" and both are under "ALSCleanerSuite". You don't need to use the name suggested by me. Just make sure the name concisely describe the test itself.